spec/tree-view-spec.js

```diff
@@ -43,4 +43,28 @@ describe('TreeView', () => {
       )
     })
   })
+
+  describe('clicking', () => {
+    it('should leave multiple entries selected on right click', () => {
+      const treeView = new TreeView({})
+      const entries = treeView.roots[0].entries
+      treeView.selectEntry(entries.children[0])
+      treeView.selectMultipleEntries(entries.children[1])
+      treeView.showMultiSelectMenu()
```

Comment:
To avoid explicitly stating this in the spec, is it possible to do a `treeView.onMouseDown` event on `entries.children[1]`?